scripts/image_loader.js

```diff
@@ -181,7 +181,7 @@ settingsLoadedEvent.addHandler(function()
             {
                 var video_id;
                 
-                if ((video_id = href.match(/i.imgur\.com\/(\w+)/i)))
+                if ((video_id = href.match(/[i.]?imgur\.com\/(\w+)/i)))
```

Comment:
I haven't tested it but shouldn't this be `(i.)?` (with parentheses instead of brackets)?  I think this breaks the original case, now you're accepting the character "i" or the character "." or nothing, but not the string "i."